extension/extension.go

```diff
@@ -77,13 +67,13 @@ func CompareVersions(version1 string, version2 string) int {
 	return 0
 }
 
-// FilterForUpdates filters `extensions` down to only the extensions that are being checked,
+// FilterForUpdates filters extensions down to only the extensions that are being checked,
 // and only the ones that we have updates for.
-func (updateRequest *UpdateRequest) FilterForUpdates(allExtensionsMap *ExtensionsMap) UpdateResponse {
-	filteredExtensions := UpdateResponse{}
+func FilterForUpdates(extensions Extensions, allExtensionsMap *ExtensionsMap) Extensions {
+	filteredExtensions := Extensions{}
```

Comment:
Similar to above- I think this is fine; it seems the old code had typedef'ed `Extensions` as `UpdateResponse`. But it's semantically the same